### PR TITLE
Add render params to collections

### PR DESCRIPTION
**Pull request by smohiudd**
This PR includes:
- renders params using [render stac extension](https://github.com/stac-extensions/render). 
- `render` and `item-assets` stac extension schema links
- various collection meta data fixes

### CODE REVIEW

**Comment by vincentsarago** [comment]
@smohiudd should we add `https://stac-extensions.github.io/render/v1.0.0/schema.json` in the stack-extension array?

File: ingestion-data/collections/CMIP245-winter-median-pr.json [hunk 1]
```diff
@@ -39,5 +39,18 @@
...
+            "bidx": 1,
+            "nodata": "nan",
+            "colormap_name": "rdbu",
+            "rescale": [
+                -60,
+                60
+            ],
+            "title": "VEDA Dashboard Render Parameters"
+        }
+    }
```
Comment (vincentsarago) [inline ingestion-data/collections/CMIP245-winter-median-pr.json:55]: I think we are missing the `assets` key

Comment (smohiudd) [reply ingestion-data/collections/CMIP245-winter-median-pr.json:55]: I don't think we currently use them for the dashboard but I can add

Comment (vincentsarago) [reply ingestion-data/collections/CMIP245-winter-median-pr.json:55]: I think `assets` is the only required key of the render extension

Comment (smohiudd) [reply ingestion-data/collections/CMIP245-winter-median-pr.json:55]: Oh right I understand it now, just read the spec again. I'll add it in there.

File: ingestion-data/collections/CMIP245-winter-median-pr.json [hunk 2]
```diff
@@ -39,5 +41,21 @@
     },
     "dashboard:is_periodic": false,
     "dashboard:time_density": "year",
-    "stac_version": "1.0.0"
+    "stac_version": "1.0.0",
+    "renders": {
+        "dashboard": {
+            "resampling": "bilinear",
+            "bidx": 1,
```
Comment (vincentsarago) [inline ingestion-data/collections/CMIP245-winter-median-pr.json:48]: FYI: right now (will be fixed in next titiler/rio-tiler version) you cannot pass `bidx=1` to titiler STAC endpoint but you need to pass `asset_bidx=cog_default|1`

Comment (vincentsarago) [reply ingestion-data/collections/CMIP245-winter-median-pr.json:48]: ref https://github.com/developmentseed/titiler/pull/724

Comment (smohiudd) [reply ingestion-data/collections/CMIP245-winter-median-pr.json:48]: Does it also need to be an array? Like this:
"asset_bidx": [1]

Comment (smohiudd) [reply ingestion-data/collections/CMIP245-winter-median-pr.json:48]: @vincentsarago looks like `bidx=1` currently works with `preview` and `map` endpoints. Will that be changing in the future?

Comment (vincentsarago) [reply ingestion-data/collections/CMIP245-winter-median-pr.json:48]: it shouldn't change 👍

**Comment by smohiudd** [comment]
@emmanuelmathot are you able to release the render stac extension so we can validate the rendering parameters?

**Comment by smohiudd** [comment]
Thanks @emmanuelmathot! How can we get the identifier link to work as well? https://stac-extensions.github.io/render/v1.0.0/schema.json

**Comment by vincentsarago** [comment]
@smohiudd I do not have admin rights to add you :-(

**Comment by emmanuelmathot** [comment]
Sorry. I think I used a wrong tag for the release. Il will fix it and add you again as maintainers.

**Review by anayeaye — APPROVED** [review]

**Pull request merged**